src/containers/List/ListWrapper.tsx

```diff
@@ -146,19 +148,21 @@ const ListWrapper: React.FC = () => {
           filterListParameters={{ userId: currentUser?.userId || 0, templateCode: type }}
           totalCount={applicationCount}
         />
-        <TableMobileHeader
-          options={columns
-            .filter((col) => col.sortName !== '')
-            .map((col) => ({
-              key: col.sortName,
-              text: col.headerName || col.sortName,
-              value: col.sortName,
-            }))}
-          sortColumn={sortQuery.sortColumn}
-          sortDirection={sortQuery.sortDirection}
-          handleSort={handleSort}
-          defaultSort="last-active-date"
-        />
+        {isMobile && (
```

Comment:
Ooops! When I modified this yesterday, I forgot to put `isMobile` condition in. 🙄

Reply:
I was actually going to ask if this was meant to be in desktop as well....